src/Integration.TeamExplorer.UnitTests/Integration.TeamExplorer.UnitTests.csproj

```diff
@@ -36,18 +37,6 @@
     <PackageReference Include="NuGet.Core" Version="2.12.0" />
     <PackageReference Include="NuGet.VisualStudio" Version="3.3.0" />
   </ItemGroup>
-
-  
-  <ItemGroup Label="Alm authentication">
-    <!-- The package from MS isn't signed so we need to reference the StrongName package
-         and conditionally set the DisableStrongName properties -->
-    <PackageReference Include="Microsoft.Alm.Authentication" Version="4.0.0.1" />
-    <PackageReference Include="StrongNamer" Version="0.0.8" />
-  </ItemGroup>
-  <PropertyGroup>
-      <DisableStrongNamer Condition="'$(SignAssembly)' != 'true'">true</DisableStrongNamer>
-  </PropertyGroup>
```

Comment:
Moved this common block into the shared properties file.

This is preparation for trying to sign the ALM assembly with our key.